apps/mull-ui/src/app/components/event-page-header/event-page-header.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretRight } from '@fortawesome/free-solid-svg-icons';
+
+import { faClock } from '@fortawesome/free-regular-svg-icons';
+
```

Comment:
I think we can have svg imports without spaces in between them

Reply:
yup, removed the space